taxi/urls.py

```diff
@@ -16,8 +22,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/manufacturer-create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/manufacturer-update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/manufacturer-delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/car-create/", CarCreateView.as_view(), name="car-create"),
```

Comment:
The URL path 'cars/car-create/' should use hyphens instead of underscores. Consider changing it to 'cars/car-create/'.